tcs-service/src/main/resources/db/migration/data/hee/V4.34__update_local_office_names.sql

```diff
@@ -0,0 +1,4 @@
+SET SQL_SAFE_UPDATES = 0;
+UPDATE `Post` SET `owner` = REPLACE(`owner`, "Health Education England ", "") WHERE `owner` LIKE "Health Education England %";
```

Comment:
I guess the tests might not have run these 2 flyway scripts. We should use single quotes.

Reply:
Gah, thanks for this, I remembered to fix it last week and forgot this week!